server/core/templates/apps_partial.html

```diff
@@ -1,3 +1,28 @@
-<main class="flex-shrink-0 form-signin">
-	<h2>Apps Go Here</h2>
+<main class="p-3 x-auto">
+	<div class="(( crate::https::ui::CSS_PAGE_HEADER ))">
+		<h2>Applications list</h2>
+    </div>
+	(% if apps.is_empty() %)
+		<h5>No linked applications available</h5>
+	(% else %)
+	<div class="row row-cols-1 row-cols-sm-2 row-cols-md-3 g-3">
+		(% for app in apps %)
+		<div class="col-md-3">
+			<div class="card text-center">
+			(% match app %)
+				(% when AppLink::Oauth2 with { name, display_name, redirect_url, has_image } %)
+				<a href="(( redirect_url ))" class="link-dark stretched-link mt-2">
+					(% if has_image %)
+						<img src="/ui/images/oauth2/(( name ))" class="oauth2-img" alt="((display_name)) icon" id="(( name ))">
```

Comment:
Mental note, check this url is cached (and our other ui urls are not)

Reply:
Yeah this doesn't seem to get cached:
```
HTTP/1.1 200 OK
content-type: image/jpeg
content-length: 14075
content-security-policy: default-src 'self'; base-uri 'self' https:; form-action 'self' https:;frame-ancestors 'none'; img-src 'self' data:; worker-src 'none'; script-src 'self' 'unsafe-eval' 'sha384-YvpcrYf0tY3lHB60NNkmXc5s9fDVZLESaAA55NDzOxhy9GkcIdslK1eN7N6jIeHz' 'sha384-ujb1lZYygJmzgSwoxRggbCHcjc0rB2XoQrxeTUQyRjrOnlCoYta87iKBWq3EsdM2' 'sha384-j55kjUt6fvgxRPzrED9pe3r179Nod1RYTUzquoKlgEnsjxHkXutHcIws8u8/seDf' 'sha384-9w+Roa3ri0y9Ouy1kgFlBIwzZkKygG1EMgpvhBvdNUG2PLMNT2hxwHL4o2bDtOfF' 'sha384-gLRkh0byHbUW4T8eGWfGzvMsvJu6Umb1JY1CPYRKzC6a2y56aj3TQYtHV4eQOisC';
x-content-type-options: nosniff
permissions-policy: fullscreen=(), geolocation=()
referrer-policy: no-referrer-when-downgrade
x-kanidm-version: 1.3.0-dev
strict-transport-security: max-age=86400
x-kanidm-opid: 6719e98f-be80-4def-9849-1babc416ff99
date: Tue, 09 Jul 2024 20:02:52 GMT
```